app/assets/stylesheets/content/_in_place_editing.sass

```diff
@@ -76,6 +76,8 @@ $inplace-edit--color--very-dark: #cacaca
     & > .inplace-edit--date-range-end-date-picker
       @extend %inline-date-picker-container-position-absolute
       right: 0
+  .ui-datepicker .ui-datepicker-calendar td a.ui-state-active
+    border-color: #008bd0 !important
```

Comment:
I'd rather use the primary color here than an arbitrary color value (which will not change among themes)